scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -250,19 +250,24 @@ We will store the PCA results in our `SingleCellExperiment` object, as we will w
 To do this, we will use the `runPCA()` function from `scater`, which performs the PCA calculations and returns a new object with the results stored in the `reducedDim` slot.
 If we wanted to, we could get the raw results as a matrix instead with `calculatePCA()` function, as we did in a previous notebook.
 
+We will also use the `ntop` argument to calculate the PCA using more genes than the default 500 with highest variance.
```

Comment:
This tripped me up when I read it

```suggestion
We will also use the `ntop` argument to calculate the PCA using 2000 genes with the highest variance.
The default is `ntop = 500`.
```

Can maybe state why that might be too low?